### PR TITLE
CASS-1799 Fixing the Priam config endpoints after they were broken du…

### PR DESCRIPTION
**Pull request by schakrovorthy**
…e to a dependency update.

### CODE REVIEW

File: priam/src/main/java/com/netflix/priam/resources/PriamConfig.java [hunk 1]
```diff
@@ -51,14 +52,17 @@ private Response doGetPriamConfig(String group, String name) {
...
                 result.put(name, value.get(name));
-                return Response.ok(result, MediaType.APPLICATION_JSON).build();
+                return Response.ok(GsonJsonSerializer.getGson().toJson(result)).build();
             } else if (name != null) {
                 result.put("message", String.format("No such structured config: [%s]", name));
                 logger.error(String.format("No such structured config: [%s]", name));
-                return Response.status(404).entity(result).type(MediaType.APPLICATION_JSON).build();
+                return Response.status(404)
+                        .entity(GsonJsonSerializer.getGson().toJson(result))
+                        .type(MediaType.TEXT_PLAIN)
```
Comment (arunagrawal84) [inline priam/src/main/java/com/netflix/priam/resources/PriamConfig.java]: Why this has to be plain text even when we convert to JSON?

File: priam/src/main/java/com/netflix/priam/resources/PriamConfig.java [hunk 2]
```diff
@@ -89,11 +92,14 @@ public Response getProperty(
...
                 result.put(name, value);
-                return Response.ok(result, MediaType.APPLICATION_JSON).build();
+                return Response.ok(GsonJsonSerializer.getGson().toJson(result)).build();
             } else {
                 result.put("message", String.format("No such property: [%s]", name));
                 logger.error(String.format("No such property: [%s]", name));
-                return Response.status(404).entity(result).type(MediaType.APPLICATION_JSON).build();
+                return Response.status(404)
+                        .entity(GsonJsonSerializer.getGson().toJson(result))
+                        .type(MediaType.TEXT_PLAIN)
```
Comment (arunagrawal84) [inline priam/src/main/java/com/netflix/priam/resources/PriamConfig.java]: Same as other.

File: priam/src/main/java/com/netflix/priam/resources/PriamConfig.java (hunk 1 above)
Comment (schakrovorthy) [reply priam/src/main/java/com/netflix/priam/resources/PriamConfig.java]: Fair point I have modified it to APPLICATION_JSON now.

File: priam/src/main/java/com/netflix/priam/resources/PriamConfig.java (hunk 2 above)
Comment (schakrovorthy) [reply priam/src/main/java/com/netflix/priam/resources/PriamConfig.java]: Fair point I have modified it to APPLICATION_JSON now.

**Pull request merged**